syntax = "proto3";

// The greeting service definition.
service Matchmaker {
  // Sends a greeting
  rpc GetMatch (QuickMatchRequest) returns (QuickMatchReply) {}
}

// The request message containing the user's name.
message QuickMatchRequest {
  string username = 1;
  bytes password = 2;
}

// The response message containing the greetings
message QuickMatchReply {
  string username = 1;
  string server = 2;
  string auth_key = 3;
  double ranking = 4;
  string response = 5;
}